docs/docs/index.md

```diff
@@ -0,0 +1,121 @@
+<!--
+ ~ SPDX-FileCopyrightText: Copyright DB Netz AG and the capella-collab-manager contributors
+ ~ SPDX-License-Identifier: Apache-2.0
+ -->
+
+# Welcome
+
+Welcome to the MBSE Docker images repository. Initially started for our [Capella Collaboration Manager](https://github.com/DSD-DBS/capella-collab-manager) to run Capella in a browser, we now offer a variety of Docker images to automate processes in the MBSE context.
+
+## Use prebuilt images from Github packages
+
+For license reasons, we are only able to provide the following prebuilt public images at this time:
+
+- [`base`](base.md)
+- [`capella/base`](capella/base.md) (without dropins or plugins)
+- [`capella/readonly`](capella/readonly.md)
+- [`capella/cli`](capella/cli.md)
+
+If you need another image, please follow the [`Build images locally`](#build-images-locally) or [`Build images in a CI/CD environment`](#build-images-in-a-cicd-environment) instructions.
+
+## Build images locally
+
+To get started, please clone this repository and include all submodules:
+
+```zsh
+git clone --recurse-submodules https://github.com/DSD-DBS/capella-dockerimages.git
+```
```

Comment:
Why do we need the `--recurse-submodules` here?

Reply:
We had a submodule (pyease) some time ago, but it doesn't exist anymore.